src/result.rs

```diff
@@ -81,6 +89,35 @@ impl QueryResultWrapper {
       .map(|f| f.name.clone())
       .collect()
   }
+
+  #[napi]
+  pub fn get_columns_specs(&self) -> Vec<MetaColumnsWrapper> {
+    self
+      .internal
+      .col_specs()
+      .iter()
+      .map(|f| {
+        MetaColumnsWrapper {
+          // ToDo: Check if rust provides such information
+          ksname: "unknown?".to_string(),
```

Comment:
Keep in mind that once the ongoing deserialization refactor is done in Rust driver (in the next release, possibly next Thursday), `QueryResult` and its API is going to change much.